README.md

```diff
@@ -1,4 +1,4 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/k8sportforward.svg?&style=shield&circle-token=158ee287ed79ce51d7cf6fcc04637c5a82428036)](https://circleci.com/gh/giantswarm/k8sportforward)
+[![CircleCI](https://circleci.com/gh/giantswarm/k8sportforward.svg?&style=shield&circle-token=a1a6d79291b52022182456a68ac420b8c793344a)](https://circleci.com/gh/giantswarm/k8sportforward)
```

Comment:
Somebody copied this without thinking. There wasn't any token. I created one. The badge should work now.